fake-vm-component-scripts/fake-vm/launch.py

```diff
@@ -10,25 +10,30 @@
 
 instances = []
 
-def fix_config_keys(source):
-    for k, v in source:
-        yield (k, v) if '.' not in k else (k.split('.')[1], v)
+config_keys = {
+    k: k.split('.')[1] for k in {
+        'configuration.name',
+        'configuration.address',
+        'configuration.login',
+        'configuration.failure',
```

Comment:
> we don't expose instance names on scriptable components

Yes, it is a known limitation

> I passed it through the configuration

So, we have an instance name (`I`), a name in the configuration (`C`) and a name in the external system (`R`). All three can be different. When instance is launched or renamed `I` is supposed to be copied to `C` manually. `C` is will be copied to `R` by scripts, if `R` changes it will affect `I` but only if organization is recreated (which is actually very unlikely in prod). Isn't it too complicated to be useful?